django/db/models/query.py

```diff
@@ -890,7 +890,7 @@ def datetimes(self, field_name, kind, order='ASC', tzinfo=None):
         else:
             tzinfo = None
         return self.annotate(
-            datetimefield=Trunc(field_name, kind, output_field=DateTimeField(), tzinfo=tzinfo),
+            datetimefield=Trunc(field_name, kind, output_field=DateTimeField(), tzinfo=tzinfo, is_dst=True),
```

Comment:
This should not default to `is_dst=True` as it will be backward incompatible.

We should allow passing it as a `datetimes` kwarg and default to `None` just like `Trunc(is_dst=None)` does and pass `is_dst=True` from the admin call if necessary. 

This should likely be in a different commit and include docs.

Reply:
Thanks @charettes.
I added the `is_dst` parameter to `datetimes` in a separate commit.